src/components/WalletModal/index.tsx

```diff
@@ -191,7 +164,9 @@ export default function WalletModal({
   const redesignFlagEnabled = redesignFlag === RedesignVariant.Enabled
   const [walletView, setWalletView] = useState(WALLET_VIEWS.ACCOUNT)
   const [lastActiveWalletAddress, setLastActiveWalletAddress] = useState<string | undefined>(account)
-  const [shouldLogWalletBalances, setShouldLogWalletBalances] = useState(false)
+  const [shouldLogUsdcBalance, setShouldLogUsdcBalance] = useState(false)
```

Comment:
can these 3 be a single state?

Reply:
i dont think it makes sense to because some of them might not be available / present for certain wallets, so we want to log them when / if a single one becomes available independent of the others

Reply:
that makes sense